test/clj/rems/test_browser.clj

```diff
@@ -282,15 +293,69 @@
                        ;; TODO could test other fields here too, e.g. title
                        [(:field/type field)
                         (:field/value field)]))))
-
             (testing "after navigating to the application view again"
               (scroll-and-click *driver* [{:css "table.my-applications"}
                                           {:tag :tr :data-row application-id}
                                           {:css ".btn-primary"}])
               (wait-visible *driver* {:tag :h1, :fn/has-text "Application"})
               (wait-page-loaded)
               (testing "check a field answer"
-                (is (= "Test name" (get-element-text *driver* field-selector)))))))))))
+                (is (= "Test name" (get-element-text *driver* description-field-selector)))))))))))
+
+(defn get-attachments
+  ([]
+   (get-attachments {:css "a.attachment-link"}))
+  ([selector]
+   (mapv (partial get-element-text-el *driver*) (query-all *driver* selector))))
+
+(deftest test-handling
+  (let [applicant "alice"
+        handler "developer"
+        form-id (test-data/create-form! {:form/fields [{:field/title {:en "description" :fi "kuvaus"}
+                                                        :field/optional false
+                                                        :field/type :description}]})
+        catalogue-id (test-data/create-catalogue-item! {:form-id form-id})
+        application-id (test-data/create-draft! applicant
+                                                [catalogue-id]
+                                                "test-handling")]
+    (test-data/command! {:type :application.command/submit
+                         :application-id application-id
+                         :actor applicant})
+    (with-postmortem *driver* {:dir reporting-dir}
+      (login-as handler)
+      (testing "handler should see todos on logging in"
+        (wait-visible *driver* :todo-applications))
+      (testing "handler should see description of application"
+        (wait-visible *driver* {:class :application-description :fn/text "test-handling"}))
+      (let [app-button {:tag :a :href (str "/application/" application-id)}]
+        (testing "handler should see view button for application"
+          (wait-visible *driver* app-button))
+        (click *driver* app-button))
```

Comment:
Should these all be `scroll-and-click`, and should it be named just `click` later?

Reply:
I think scroll-and-click failed for some things in the floating action div. I guess this specific instance could be scroll-and-click. I'll try it out.